gbi.h

```diff
@@ -3073,11 +3113,53 @@ typedef union {
 
 
 /**
- * Insert values into Vertices
- *
- * vtx = vertex number 0-55
- * where = which element of point to modify (byte offset into vertex)
- * num   = new value (32 bit)
+ * @brief You can use this macro to modify certain sections of a vertex after it has been sent to the RSP (by the gSPVertex macro).
+ * 
+ * This is an advanced macro. You need a good understanding of how vertices work in the RSP microcode before you use this macro (refer to gSPVertex). The range of vtx has been changed with as the vertex cache increased. Its range is now the same as that for gSPVertex.
```

Comment:
Please remove `The range of vtx has been changed with as the vertex cache increased. Its range is now the same as that for gSPVertex.`, this is again referring to old limitations pre-F3DEX.